APCAppCore/APCAppCore/UI/TabBarControllers/Profile/APCProfileViewController.m

```diff
@@ -32,22 +32,36 @@
 // 
  
 #import "APCProfileViewController.h"
-#import "APCAppCore.h"
+#import "APCPermissionsManager.h"
+#import "APCSharingOptionsViewController.h"
+#import "APCLicenseInfoViewController.h"
+#import "APCIntroVideoViewController.h"
+#import "APCSignUpPermissionsViewController.h"
+#import "APCChangePasscodeViewController.h"
+#import "APCWithdrawCompleteViewController.h"
+#import "APCWebViewController.h"
+#import "APCSettingsViewController.h"
+#import "APCSpinnerViewController.h"
 #import "APCTableViewItem.h"
-#import "APCAppDelegate.h"
+#import "APCAppDelegate.h"				// should be removed
```

Comment:
If this import should be removed, why hasn't it? An explanation would be useful.